build/includes/google-cloud.mk

```diff
@@ -53,28 +53,28 @@ clean-gcloud-test-cluster: $(ensure-build-image)
 gcloud-e2e-test-cluster: GCP_PROJECT ?= $(shell $(current_project))
 gcloud-e2e-test-cluster: $(ensure-build-image)
 gcloud-e2e-test-cluster:
-	$(MAKE) terraform-init DIRECTORY=e2e
+	$(MAKE) terraform-init BUCKET=$(GCP_PROJECT)-e2e-infra-bucket-tfstate PREFIX=terraform/state DIRECTORY=e2e
 	docker run --rm -it $(common_mounts) $(DOCKER_RUN_ARGS) $(build_tag) bash -c 'cd $(mount_path)/build/terraform/e2e && \
       	terraform apply -auto-approve -var project="$(GCP_PROJECT)"'
 
 # Deletes the gcloud e2e cluster and cleanup any left pvc volumes
 clean-gcloud-e2e-test-cluster: GCP_PROJECT ?= $(shell $(current_project))
 clean-gcloud-e2e-test-cluster: $(ensure-build-image)
 clean-gcloud-e2e-test-cluster:
-	$(MAKE) terraform-init DIRECTORY=e2e
+	$(MAKE) terraform-init BUCKET=$(GCP_PROJECT)-e2e-infra-bucket-tfstate PREFIX=terraform/state DIRECTORY=e2e
 	$(DOCKER_RUN) bash -c 'cd $(mount_path)/build/terraform/e2e && terraform destroy -var project=$(GCP_PROJECT) -auto-approve'
 
 # Creates a gcloud cluster for prow
 gcloud-prow-build-cluster: GCP_PROJECT ?= $(shell $(current_project))
 gcloud-prow-build-cluster: $(ensure-build-image)
 gcloud-prow-build-cluster:
-	$(MAKE) terraform-init DIRECTORY=prow
+	$(MAKE) terraform-init BUCKET=$(GCP_PROJECT)-prow-infra-bucket-tfstate PREFIX=terraform/state DIRECTORY=prow
```

Comment:
maybe not strictly necessary here, but given that we are at an inflection point of changing the e2e infrastructure _and_ I don't think any of us understands if the Prow configuration is even working, should we just instead turn down Prow? I am afraid that if we continue as is we'll give someone a false sense of "maybe it works" when I feel like it has reached the point of bitrot.

Reply:
I have no objections to removing the prow cluster.

Reply:
> I have no objections to removing the prow cluster.

Who are you, and what have you done with the real @roberthbailey ? 😁